src/components/implementation/no_interface/vkernel/vk_api.c

```diff
@@ -113,62 +114,70 @@ vk_virtmem_alloc(struct vms_info *vminfo, struct vkernel_info *vkinfo,
 }
 
 void
-vk_shmem_alloc(struct vms_info *vminfo, struct vkernel_info *vkinfo, 
-	       unsigned long shm_ptr, unsigned long shm_sz)
+vk_shmem_alloc(struct vms_info *vminfo, struct vkernel_info *vkinfo, unsigned long shm_ptr, unsigned long shm_sz)
 {
 	vaddr_t src_pg = (shm_sz * vminfo->id) + shm_ptr, dst_pg, addr;
 
 	assert(vminfo && vminfo->id == 0 && vkinfo);
 	assert(shm_ptr == round_up_to_pgd_page(shm_ptr));
 
-	for (addr = shm_ptr ; addr < (shm_ptr + shm_sz) ; addr += PAGE_SIZE, src_pg += PAGE_SIZE) {
+	for (addr = shm_ptr; addr < (shm_ptr + shm_sz); addr += PAGE_SIZE, src_pg += PAGE_SIZE) {
 		/* VM0: mapping in all available shared memory. */
 		src_pg = (vaddr_t)cos_page_bump_alloc(&vkinfo->shm_cinfo);
 		assert(src_pg && src_pg == addr);
 
 		dst_pg = cos_mem_alias(&vminfo->shm_cinfo, &vkinfo->shm_cinfo, src_pg);
 		assert(dst_pg && dst_pg == addr);
-	}	
+	}
 
 	return;
 }
 
 void
-vk_shmem_map(struct vms_info *vminfo, struct vkernel_info *vkinfo, 
-	     unsigned long shm_ptr, unsigned long shm_sz)
+vk_shmem_map(struct vms_info *vminfo, struct vkernel_info *vkinfo, unsigned long shm_ptr, unsigned long shm_sz)
 {
 	vaddr_t src_pg = (shm_sz * vminfo->id) + shm_ptr, dst_pg, addr;
 
 	assert(vminfo && vminfo->id && vkinfo);
 	assert(shm_ptr == round_up_to_pgd_page(shm_ptr));
 
-	for (addr = shm_ptr ; addr < (shm_ptr + shm_sz) ; addr += PAGE_SIZE, src_pg += PAGE_SIZE) {
+	for (addr = shm_ptr; addr < (shm_ptr + shm_sz); addr += PAGE_SIZE, src_pg += PAGE_SIZE) {
 		/* VMx: mapping in only a section of shared-memory to share with VM0 */
 		assert(src_pg);
 
 		dst_pg = cos_mem_alias(&vminfo->shm_cinfo, &vkinfo->shm_cinfo, src_pg);
 		assert(dst_pg && dst_pg == addr);
-	}	
+	}
 
 	return;
 }
 
 thdcap_t
 dom0_vio_thdcap(unsigned int vmid)
-{ return DOM0_CAPTBL_SELF_IOTHD_SET_BASE + (CAP16B_IDSZ * (vmid-1)); }
+{
```

Comment:
This is annoying, because I can make it accept this:
```
 thdcap_t
 dom0_vio_thdcap(unsigned int vmid) { return DOM0_CAPTBL_SELF_IOTHD_SET_BASE + (CAP16B_IDSZ * (vmid-1)); }
```
But not the way it is :(